docker/google-cloud-securitycenter/Dockerfile

```diff
@@ -0,0 +1,8 @@
+
+FROM demisto/python3:3.8.6.14516
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git \
+  && pip install --no-cache-dir -r requirements.txt \
+  && apk del .build-dependencies
```

Comment:
```suggestion
RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git \ gcc
  && pip install --no-cache-dir -r requirements.txt \
  && apk del .build-dependencies
```

Reply:
gcc is installed as part of python3-dev. The problem the package is failing to compile not a missing compiler.